<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>RzxIntro</class>
 <widget class="QDialog" name="RzxIntro" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Welcome</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" colspan="2" >
    <widget class="QListWidget" name="listIcons" >
     <property name="minimumSize" >
      <size>
       <width>330</width>
       <height>74</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>330</width>
       <height>74</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="gridSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="viewMode" >
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="cbIcons" />
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="cbLanguage" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lblIcon" >
     <property name="text" >
      <string>Select your icon theme:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="lblLanguage" >
     <property name="text" >
      <string>Select your language:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>cbLanguage</tabstop>
  <tabstop>cbIcons</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>listIcons</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
